src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.tsx

```diff
@@ -82,173 +82,170 @@ export type QueryWrapperPlotNavProps = SearchParams &
   Operator &
   OwnProps
 
-const QueryWrapperPlotNav: React.FunctionComponent<QueryWrapperPlotNavProps> =
-  props => {
```

Comment:
Just changed `props` to `(props: QueryWrapperPlotNavProps)` because that causes props to show up in Styleguidist